tests/lava/magma/core/model/py/test_type.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+import unittest
+from lava.magma.core.model.py.type import LavaPyType
+import numpy as np
+
+
+class TestLavaPyType(unittest.TestCase):
+
+    def test_lavapytype_constructor(self):
+        """Check if LavaPyType is correctly instantiated"""
+        lava_py_type = LavaPyType(cls=None, d_type=None)
+
+        # Check if correct instance is created.
+        self.assertIsInstance(lava_py_type, LavaPyType)
+
+    def test_validate_precision_input_warning(self):
+
```

Comment:
Some docstring briefly explaining what the unittest checks would be good. (Also for the other tests)

Reply:
Addressed in 8092d313c76b4e45680cfc4aee491210ae60321a.